src/components/pill-selector-dropdown/PillSelectorDropdown.js

```diff
@@ -38,6 +38,8 @@ type Props = {
     getPillImageUrl?: (data: { id: string, [key: string]: any }) => string | Promise<?string>,
     /** Passed in by `SelectorDropdown` for accessibility */
     inputProps: Object,
+    /** Option to enable dynamic positioning with popper */
+    isPositionDynamic?: boolean,
```

Comment:
Should this be a required prop (like some of the other boolean props)? If not, should the default value be `undefined` instead?

Reply:
this came up from this PR: https://github.com/box/box-ui-elements/pull/3262 where `SelectorDropdown` defaults to false

we don't want to make it required but what's the advantage of making it `undefined`?

Reply:
Ah I was thinking so we don't have `isPositionDynamic={false}` added to the snapshots and the prop could be added to the component with just `isPositionDynamic` instead of `isPositionDynamic={true}`. It's not a super strong suggestion, more of a question. WDYT?

Reply:
I think leaving it undefined makes sense since it's just a passthrough prop to `SelectorDropdown`. and then `SelectorDropdown` can handle the default value. that way we don't need to update in two places in the future.

Reply:
Makes sense

Reply:
HALF A BOOLEAN??